rpc/v2/environments/environments.proto

```diff
@@ -132,6 +132,7 @@ message EnvironmentConfiguration {
   optional string directory = 2;
   optional string remote = 3;
   optional string base = 4;
+  optional SCM scm = 5;
```

Comment:
this makes reasoning about which icon to show in the ui easier

Reply:
nice